File: 709.to-lower-case.cpp
/*
 * @lc app=leetcode id=709 lang=cpp
 *
 * [709] To Lower Case
 */

// @lc code=start
class Solution
{
public:
    string toLowerCase(string str)
    {
        for (char &c : str)
        {
            if (c >= 'A' && c <= 'Z')
                c += 32;
        }
        return str;
    }
};
// @lc code=end

File: 709.to-lower-case.js
/*
 * @lc app=leetcode id=709 lang=javascript
 *
 * [709] To Lower Case
 */

// @lc code=start
/**
 * @param {string} str
 * @return {string}
 */
var toLowerCase = function (str) {
    let res = "";

    for (let i = 0; i < str.length; i++) {
        let ascii = str.charCodeAt(i);
        if (ascii >= 65 && ascii <= 90) res += String.fromCharCode(ascii + 32);
        else res += str.charAt(i);
    }

    return res;
};
// @lc code=end

